generate-activities-types.sh

```diff
@@ -1 +1 @@
-npx openapi-typescript https://activities-api-dev.prison.service.justice.gov.uk/v3/api-docs > server/@types/activitiesAPI/index.d.ts
+npx openapi-typescript http://localhost:8080/v3/api-docs > server/@types/activitiesAPI/index.d.ts
```

Comment:
This an accidental one?

Reply:
sure is - had to temporarily do that to generate the types